.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "bracketSpacing": true,
+  "parser": "typescript"
+}
```

Comment:
I'd love to remove this config at some point and just use whatever default Prettier uses.

Reply:
I just checked and removing this config would replace all single quotes with doubles. If you don't mind, I can do this in this PR

Reply:
Let's do it in a separate PR. I think we want to upgrade to 3.x as well.